todo_app/SeleniumTest/Tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+# import  pytest
```

Comment:
We're not using this file or any others with just commented out code, so let's delete it